docs/content/action/output.md

```diff
@@ -0,0 +1,31 @@
+---
+navtitle: Output
+sort: 3
+---
+
+# Action output
+
+Action output is the result returned by Nikita's actions. It is defined by the [action handler](/current/action/handler#return) enriched with the default values.
+
+- `status` - a boolean value of the [action status](/current/usages/status).
+- `log` - an array of objects with [logging](/current/usages/logging_debugging) information of the action execution.
+
+Optionally, enriching the handler return value can be disabled with the [`raw_output` metadata](/current/metadata/raw_output).
```

Comment:
Use the `raw_output` metadata to instruct plugins to not alter the output (this deserve its own section with an example)

Reply:
I am referencing  to page where will be this example